### DISCUSSION TITLE
How to set up a development environment for contributing?

### DISCUSSION THREAD

**Discussion by jgarte** [pytest-dev/pytest#10236] [Q&A] [answered]
Hi, how do I set up a development environment for contributing? Should I use poetry?

**Comment by nicoddemus** [comment, marked as answer]
Hi @jgarte,

Thanks for the interest, please take a look at the [contributing docs](https://github.com/pytest-dev/pytest/blob/main/CONTRIBUTING.rst#preparing-pull-requests).

**Reply by jgarte** [reply]
Thanks for pointing me to those docs!
